version/version.go

```diff
@@ -0,0 +1,50 @@
+package version
+
+import (
+	_ "embed"
+	"fmt"
+	"strings"
+)
+
+var (
+	// The git commit that was compiled. These will be filled in by the
+	// compiler.
+	GitCommit string
+
+	// The next version number that will be released. This will be updated after every release
+	// Version must conform to the format expected by github.com/hashicorp/go-version
+	// for tests to work.
+	// A pre-release marker for the version can also be specified (e.g -dev). If this is omitted
+	// then it means that it is a final release. Otherwise, this is a pre-release
+	// such as "dev" (in development), "beta", "rc1", etc.
+	//go:embed VERSION
+	fullVersion string
+
+	Version, VersionPrerelease, _ = strings.Cut(fullVersion, "-")
```

Comment:
Is there a reason to omit the `go-version` check?

Reply:
I re-added the version check. Do you know what calls the `init` function?

Reply:
The `init` function is called automatically upon import of the package (see e.g. https://www.digitalocean.com/community/tutorials/understanding-init-in-go), so in our case at the earliest stage of any `terraform-ls` command. This ensures that if it panics, then it panics very loudly and early.